controllers/employees.controller.js

```diff
@@ -48,13 +48,16 @@ exports.postItem = async (req, res) => {
 exports.putItem = async (req, res) => {
   const { firstName, lastName, department } = req.body;
   try {
-    const employee = await Employee.findById(req.params.id);
+    const employee = await Employee.updateOne(
+      { _id: req.params.id},
+      { $set: { firstName: firstName, lastName: lastName, department: department}}
+    );
     if (employee) {
-      await Employee.updateOne(
-        {_id: req.params.id},
-        { $set: {firstName, lastName, department} }
-      );
-      res.json({ message: 'OK' });
+        employee.firstName = firstName;
+        employee.lastName = lastName;
+        employee.department = department;
+        await employee.save();
+        res.json ({message: 'OK' });
```

Comment:
Nie rozumiemy się :) Omówimy to na spotkaniu w takim razie. Tutaj przywróć to co było i tak zostawmy - będzie ok